ownCloud/Bookmarks/BookmarkViewController.swift

```diff
@@ -536,7 +536,26 @@ class BookmarkViewController: StaticTableViewController {
 					} else {
 						OnMainThread {
 							hudCompletion({
-								weakSelf.presentingViewController?.dismiss(animated: true, completion: nil)
+								if issue != nil {
+									self?.bookmark?.authenticationData = nil
+
+									let issuesViewController = ConnectionIssueViewController(displayIssues: issue?.prepareForDisplay(), completion: { [weak self] (response) in
```

Comment:
`[weak self]` is here not needed, because you already using `weakself` from above

Reply:
That `[weak self]` is to make the reference to `self` weak in the `ConnectionIssueViewController` completion block. `weakSelf` above is actually a strong reference to self derived from a previous `[weak self]`. I'll change the naming to make this less confusing.

Reply:
Changed the naming in the latest commit.